docs/howto/operate/new-cluster/aws.md

```diff
@@ -1,20 +1,15 @@
-# Add a new hub in a AWS kops-based cluster
```

Comment:
For some reason `git` thinks that I have actually _copied_ the old `aws.md` file to this new space, and then _removed_ some stuff...but this is just me removing the _section_ on clusters and pasting into a new file